openvdb_houdini/openvdb_houdini/SOP_OpenVDB_Merge.cc

```diff
@@ -0,0 +1,682 @@
+            "point",     "Nearest",
+            "linear",    "Linear",
+            "quadratic", "Quadratic"
+        })
+        .setTooltip(
+            "Specify the type of interpolation to be used when\n"
+            "resampling one VDB to match the other's transform.")
+        .setDocumentation(
+            "The type of interpolation to be used when resampling one VDB"
+            " to match the other's transform\n\n"
+            "Nearest neighbor interpolation is fast but can introduce noticeable"
+            " sampling artifacts.  Quadratic interpolation is slow but high-quality."
+            " Linear interpolation is intermediate in speed and quality."));
+
+    parms.beginSwitcher("Group1");
+
+    parms.addFolder("Merge Operation");
+
+    parms.add(hutil::ParmFactory(PRM_STRING, "op_fog", "Fog VDBs")
+        .setDefault("add")
+        .setChoiceListItems(PRM_CHOICELIST_SINGLE, {
+            "none",             "None            ",
+            "add",              "Add"
+        })
+        .setTooltip("Merge operation for Fog VDBs.")
+        .setDocumentation(
+            "Merge operation for Fog VDBs.\n\n"
+            "None:\n"
+            "   Add grids to the output of the SOP without merging them.\n\n"
+            "Add:\n"
+            "    Generate the sum of all fog VDBs with same collation.\n\n"));
+
+    parms.add(hutil::ParmFactory(PRM_STRING, "op_sdf", "SDF VDBs")
+        .setDefault("sdfunion")
+        .setChoiceListItems(PRM_CHOICELIST_SINGLE, {
+            "none",             "None            ",
+            "sdfunion",         "SDF Union",
+            "sdfintersect",     "SDF Intersect"
+        })
+        .setTooltip("Merge operation for SDF VDBs.")
+        .setDocumentation(
+            "Merge operation for SDF VDBs.\n\n"
+            "None:\n"
+            "   Add grids to the output of the SOP without merging them.\n\n"
+            "SDF Union:\n"
+            "    Generate the union of all signed distance fields with same collation.\n\n"
+            "SDF Intersection:\n"
+            "    Generate the intersection of all signed distance fields with same collation.\n\n"));
+
+    parms.endSwitcher();
+
+    // Register this operator.
+    hvdb::OpenVDBOpFactory("VDB Merge", SOP_OpenVDB_Merge::factory, parms, *table)
+#ifndef SESI_OPENVDB
+        .setInternalName("DW_OpenVDBMerge")
+#endif
+        .addInput("VDBs To Merge")
+        .setMaxInputs()
+        .setVerb(SOP_NodeVerb::COOK_INPLACE, []() { return new SOP_OpenVDB_Merge::Cache; })
+        .setDocumentation("\
+#icon: COMMON/openvdb\n\
+#tags: vdb\n\
+\n\
+\"\"\"Merge VDB SDF grids.\"\"\"\n\
+\n\
+@overview\n\
+\n\
+Merges VDB SDF grids.\n\
+\n\
+Unlike the VDB Combine SOP, it provides a multi-input so can merge more than two grids without needing an \
+additional merge SOP. \n\
+Float and double VDB SDF grids can be merged, all other VDB grids are added to the output without merging. \n\
```

Comment:
Again, i don't see it as "adding to the output" so much as the node leaving them unchanged.  (Technically, adding to  the output is what we are doing, but people have many mental models of what actually happens :>)